### PR TITLE
🚨 Cover subprocess.py on mypy

### PR DESCRIPTION
**Pull request by Kludex**

### CODE REVIEW

File: scripts/check [hunk 1]
```diff
@@ -10,6 +10,6 @@ set -x
 
 ${PREFIX}black --check --diff --target-version=py36 $SOURCE_FILES
 ${PREFIX}flake8 $SOURCE_FILES
-${PREFIX}mypy
+${PREFIX}mypy --show-error-codes
```
Comment (Kludex) [inline scripts/check:13]: I forgot to remove this. Should I? @euri10

**Comment by euri10** [comment]
actually I like it, I like the --pretty flag also, and we could use
--warn-unused-ignores since this is an incremental work and if we ignore
stuff one day maybe in the future that would not be true anymore

File: scripts/check [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh -e
 
 export PREFIX=""
-if [ -d 'venv' ] ; then
```
Comment (euri10) [inline scripts/check]: seems unrelated

**Review by euri10 — APPROVED** [review]
small glitch in the check script but lgtm otherwise, approved

File: scripts/check [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
+if [ -d 'venv' ]; then
```
Comment (Kludex) [inline scripts/check]:
```suggestion
if [ -d 'venv' ] ; then
```

File: scripts/check (hunk 2 above)
Comment (Kludex) [reply scripts/check]: fixed
